src/goCheck.ts

```diff
@@ -46,6 +51,66 @@ export function notifyIfGeneratedFile(this: void, e: vscode.TextDocumentChangeEv
 	}
 }
 
+function notifyOnBuildTagMismatch(ctx: any, e: vscode.TextDocumentChangeEvent) {
+	if (ctx.globalState.get('ignoreBuildTagMismatchWarning') === true) {
+		return;
+	}
+
+	const fileBuildTags = [];
+	for (let i = 0; i < e.document.lineCount; i++) {
+		const line = e.document.lineAt(i);
+		if (line.isEmptyOrWhitespace) {
+			break;
```

Comment:
Say I have a file with a line comment preceding the build constraing
```go
// my comment

// +build hello
```

This is a valid use of build constraints, but you will end up exiting the loop based on the current code.

Based on the [docs for build constraints](https://golang.org/pkg/go/build/#hdr-Build_Constraints), line comments and empty lines can preceed the build constraint